src/RA_StringUtils.h

```diff
@@ -227,21 +240,21 @@ class StringBuilder
     }
 
     template<typename T, typename... Ts>
-    void Append(T arg, Ts... args)
+    void Append(const T&& arg, const Ts&&... args)
     {
         Append(arg);
-        Append(std::forward<Ts>(args)...);
+        Append(std::forward<const Ts>(args)...);
```

Comment:
These forwards can't use `const`. You can't forward a `const`.

Reply:
I think this is ok. Because the `const` is not part of `Ts`, removing it causes the following error:
```
2>e:\source\raintegration\src\ra_stringutils.h(246): error C2665: 'std::forward': none of the 2 overloads could convert all the argument types
```

See also https://stackoverflow.com/questions/8283616/why-doesnt-const-int-ci-2-stdforwardintci-work-and-how-to-fix-wor

> You can forward less cv-qualified expressions to more cv-qualified expressions.